[First Timers Only] Fix JUnit 5 tests should be package-private issue

### Note: This issue is to be picked up by first time contributors to this project only. If it is not picked up after ~7 days, it will be opened up for anyone to take.


## Task Context
<!---
A clear and concise description of what the task is.
-->
The Codacy Static Code Analysis system is reporting a Junit 5 package-private issue(s). The reported issue(s) can be found at:

https://app.codacy.com/gh/jmakhack/hydrate-reminder/issues?bid=22863237&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6WyJKYXZhIl19LHsiaWQiOiJDYXRlZ29yeSIsInZhbHVlcyI6WyJDb2RlU3R5bGUiXX0seyJpZCI6IkxldmVsIiwidmFsdWVzIjpbXX0seyJpZCI6IlBhdHRlcm4iLCJ2YWx1ZXMiOlsiMzU5NzAiXX0seyJpZCI6IkF1dGhvciIsInZhbHVlcyI6W119XQ==

## Acceptance Criteria / Goals
<!---
A list of acceptance criteria and/or goals that need to be fulfilled
in order to consider this task completed.
-->
The goal of this task is to fix the issue(s) reported by Codacy.

## Additional Notes
<!---
Add any other notes regarding the task if needed.
-->
After opening a pull request, the Codacy Static Code Analysis will automatically run. It should report that the fixed issue(s) if this task is completed successfully.

Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[<img src="https://img.shields.io/discord/1023562225790767175?color=%20%237289DA&label=discord&logo=discord&logoColor=%20%237289DA&style=for-the-badge" />](https://discord.gg/RTgxfFW9mS)

`I have made the changes but I am not able to run the files locally. 
`` org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':test' ``
`` Caused by: org.gradle.api.tasks.testing.TestExecutionException: No tests found for given includes: [com.hydratereminder.command.total.TotalCommandHandlerTest](--tests filter) ``

I have searched through stack overflow and tried one of the solutions, any inputs on this? I have set JAVA_HOME to 1.8 by the way

> I have made the changes but I am not able to run the files locally. `org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':test'` `Caused by: org.gradle.api.tasks.testing.TestExecutionException: No tests found for given includes: [com.hydratereminder.command.total.TotalCommandHandlerTest](--tests filter) ` I have searched through stack overflow and tried one of the solutions, any inputs on this @jmakhack ? I have set JAVA_HOME to 1.8 by the way

I'm not sure what the issue is just from that error message alone. Did you follow the steps in [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) such as setting the `-ea` VM option? Running the project through those steps should also run the unit tests automatically as well.

If you can't get the tests running locally, you can also just open a pull request and the tests will be automatically run on our pipeline.

--- a/src/test/java/com/hydratereminder/command/total/TotalCommandHandlerTest.java
+++ b/src/test/java/com/hydratereminder/command/total/TotalCommandHandlerTest.java
@@ -15,7 +15,7 @@ import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
-public class TotalCommandHandlerTest {
+class TotalCommandHandlerTest {
 
     @Mock
     private transient ChatMessageSender chatMessageSender;

